SafeguardSessions/TROUBLESHOOTING.md

```diff
@@ -0,0 +1,92 @@
+## Troubleshooting
+
+If an error occurs during the data fetching process, you can check the **Info** table to see a descriptive message about the error type. In this case, the **Sessions** table will contain the exact error record that you can inspect if you go to **Home > Transform data** and select the **Sessions** table.
+

```

Comment:
This case above, will only occur when loading data and the error occurs on the first data request. If you run into an error in the middle of a data load, then you will get a pop up in that case. And if this error is not temporary, then on reload we will get the Info and Sessions table that contains the " extracted " error.